packages/mail/MandrillInterface.js

```diff
@@ -26,6 +26,9 @@ const MandrillInterface = ({ logger }) => {
         })
       })
     },
+    isUsable () {
+      return true
```

Comment:
```suggestion
      return MANDRILL_API_KEY && MANDRILL_API_KEY.length
```
then [checkEnv](https://github.com/orbiting/backends/pull/359/files#diff-00ad47e61ae2ecc80a14cd4c2ff9fbfbR10) could be removed